core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -570,18 +572,30 @@ private boolean applyRename(RenameEntry entry) {
     mInodeStore.addChild(newParent, inode);
     inode.setParentId(newParent);
     mInodeStore.writeInode(inode);
-    updateLastModified(oldParent, entry.getOpTimeMs());
-    updateLastModified(newParent, entry.getOpTimeMs());
+    updateLastModifiedAndChildCount(oldParent, entry.getOpTimeMs(), -1);
+    updateLastModifiedAndChildCount(newParent, entry.getOpTimeMs(), 1);
```

Comment:
Should we do a special case for `oldParent == newParent`?

Reply:
good call, updated

Reply:
I think we still need to update the last modified time once?

Reply:
updated